toolkit/imageconfigs/core-efi-aarch64.json

```diff
@@ -47,11 +47,15 @@
             "PackageLists": [
                 "packagelists/hyperv-packages.json",
                 "packagelists/core-packages-image-aarch64.json",
-                "packagelists/cloud-init-packages.json"
+                "packagelists/cloud-init-packages.json",
+                "packagelists/selinux.json"
```

Comment:
Let's not include selinux.json or the KernelCommandLine option in this imageconfig to keep it in line with the changes from #2313.